### ISSUE TITLE
validator not works for repeated type

### ISSUE THREAD

**Issue by vtolstov** [mwitkow/go-proto-validators#50]
```
repeated string name = 1 [(validator.field) = {msg_exists : true}];
```
when build:
```
item declared and not used
```
validator code contains this loop:
```
 for _, item := range this.Name {
 }
```

**Comment by wuxinwei** [comment]
@vtolstov I have the some problem, did you try to solve this issue?

**Comment by vtolstov** [comment]
i switch to another proto validator https://github.com/lyft/protoc-gen-validate

**Comment by fho** [comment]
@vtolstov 
To ensure that the list is not empty you can use:
` repeated string name = 1 [{repeated_count_min: 1}];`

Despite that `msg_exists` should not generate broken code for `repeated` type. It must be a bug.

**Comment by Helcaraxan** [comment]
This issue has now been fixed via #66.
